anthos-bm-gcp-terraform/docs/login.md

```diff
@@ -82,3 +85,137 @@ You can copy this token and use it to login via the Cloud Console.
     (click image to enlarge)
   </em>
 </p>
+
+---
+
+### Login with Google Cloud Identity
+
+Instead of creating a Kubernetes Service Account Token, you can use the email
+identity associated with your Google Cloud Platform account to `Login` to the
+cluster.
+
+<p align="center">
+  <img src="images/gcp_account_login.png" width="600">
+  <em>
+    </br>
+    (click image to enlarge)
+  </em>
+</p>
+
+There are two scenarios for using this method:
+  - [Configure a new cluster for Google Cloud Identity Login](#configure-a-new-cluster-for-google-cloud-identity-login)
+  - [Configure an existing cluster for Google Cloud Identity Login](#configure-an-existing-cluster-for-google-cloud-identity-login)
+
+---
+#### Configure a new cluster for Google Cloud Identity Login
+A new cluster can be configured to allow `Login` using GCP Accounts by
+setting the `spec.clusterSecurity.authorization.clusterAdmin.gcpAccounts`
+field in the cluster configuration. This field accepts a list of GCP Accounts
+that must be allowed to `Login` to the cluster. Upon creation of the cluster
+you should be able to `Login` using any of the provided GCP Accounts. If using
+the Terraform script from this repository then you can simply set the
+[`gcp_login_accounts`](/anthos-bm-gcp-terraform/variables.tf#L181) variable with
+a list of GCP Accounts.
+
+```sh
```

Comment:
where does one edit the cluster config?

Reply:
So if using the terraform script then they have to set the terraform variable before running the terraform script. This will automatically be filled into the generated cluster configuration file - which is copied into the admin workstation. I have added a note about it. 

If someone is creating there own clusters without using this terraform script then, we wouldn't have control over where the script is and how to control it. 

Also I am checking if we can actually update this field once the cluster is already created. If that is possible then I will update this readme with how-to. In that case I will point them to where to find the cluster configuration